tests/unit/h/services/jwt_test.py

```diff
@@ -54,7 +54,16 @@ def test_decode_token_with_invalid_jwt(self, service, jwt):
                 sentinel.token, sentinel.key_set_url, sentinel.algorightms
             )
 
-    @pytest.fixture(autouse=True)
+    def test_encode_decode_idinfo(self, service):
+        provider = "orcid"
+        info = {"id": "test_id"}
+
+        encoded_info = service.encode_idinfo(provider, info)
+        decoded_info = service.decode_idinfo(provider, encoded_info)
+
+        assert decoded_info == info
```

Comment:
TODO: Add tests for all the ways the token should fail to decode (invalid key, expired token, etc).